### DISCUSSION TITLE
Welcome to assistant-base Discussions!

### DISCUSSION THREAD

**Discussion by danielkornev** [deeppavlov/dream#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our DeepPavlov community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by officialabd** [comment]
How can I fine-tune/retrain the DeepPavLov Dream models with my own datasets? (I looked into the DeepPavLov Dream documentation but nothing was about training or fine-tuning)
